core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -935,6 +942,34 @@ public static long defaultMaxClientConnectionAgeMillis() {
         return DEFAULT_MAX_CLIENT_CONNECTION_AGE_MILLIS;
     }
 
+    /**
+     * Returns the default server-side graceful connection shutdown drain duration in microseconds.
+     * If the value of this flag is greater than {@code 0}, a connection shutdown will have a drain period
+     * when client will be notified about the shutdown, but in flight requests will still be accepted.
+     *
+     * <p>The default value of this flag is {@link #DEFAULT_DEFAULT_CONNECTION_DRAIN_DURATION_MICROS}.
```

Comment:
@trustin had to change this from `@value` to `@link` after applying the suggested change with `TimeUnit.SECONDS.toMicros(1)` - let me know if that looks good to you.

Reply:
Ahhh, forgot about `@value` completely. Sorry. Let's keep the initial version that uses `{@value}`.